app/admin/user.rb

```diff
@@ -130,6 +130,9 @@
               column :project_name do |allocation|
                 allocation.project.name
               end
+              column :hourly_rate do |allocation|
+                allocation.decorate.hourly_rate
```

Comment:
Use the helper of active admin to decorate the resource instead of calling here, if you have another decorated method, this would be called every time

Reply:
Refactored on d854b1634a668afc8461d214596a7f99ca3eef46